### DISCUSSION TITLE
Feature Request: NDI Bridge on OBS

### DISCUSSION THREAD

**Discussion by juniper275** [DistroAV/DistroAV#945] [Ideas]
Hi,
it would be very useful to have the NDI bridge functionality on OBS especially on linux (client and server)

**Comment by Trouffman** [comment]
Can you describe some of the use-cases?

Because NDI Bridge is only part of the Advanced SDK and HX3 certified devices.

There might be some really cool use-case that justify the time on it.

Thanks

**Reply by juniper275** [reply]
Hi,

simple, my use case: https://ndi.video/tools/ndi-bridge/

two NDI PTZ camera on a remote city site linked to a windows workstation with ndi bridge app (of ndi core 6 now) acting as server

one workstation on another city with ndi bridge app receiving NDI PTZ camera through ndi bridge app for remote director on OBS (much better than working with RDP)

NDI Bridge works also bidirectional, all NDI bridge node are shared between the node

On linux this is impossible at the moment (there isn't ndi bridge for linux), and this is my case... i hope obs can add ndi bridge "client" inside

Thanks

**Reply by Trouffman** [reply]
So you expect the OBS-ndi plugin to act as a bridge itself?

This way a camera could connect directly (via bridge) into OBS ?

**Reply by juniper275** [reply]
Yes, it would be wonderful!

yes the bridge application allows you to receive remote ndi flows (today available only for windows)

if a plugin allowed the same thing on OBS it would be perfect

**Reply by juniper275** [reply]
https://ndi.video/tools/ndi-bridge/
